freqtrade/templates/sample_strategy.py

```diff
@@ -124,87 +124,131 @@ def populate_indicators(self, dataframe: DataFrame, metadata: dict) -> DataFrame
 
-        # # Awesome oscillator
+        # # Awesome Oscillator
         # dataframe['ao'] = qtpylib.awesome_oscillator(dataframe)
 
-        # # Commodity Channel Index: values Oversold:<-100, Overbought:>100
+        # # Keltner Channel
+        # keltner = qtpylib.keltner_channel(dataframe)
+        # dataframe["kc_upperband"] = keltner["upper"]
+        # dataframe["kc_lowerband"] = keltner["lower"]
+        # dataframe["kc_middleband"] = keltner["mid"]
+        # dataframe["kc_percent"] = (
+        #     (dataframe["close"] - dataframe["kc_lowerband"]) /
+        #     (dataframe["kc_upperband"] - dataframe["kc_lowerband"])
+        # )
+        # dataframe["kc_width"] = (
+        #     (dataframe["kc_upperband"] - dataframe["kc_lowerband"]) / dataframe["kc_middleband"]
+        # )
+
+        # # Ultimate Oscillator
+        # dataframe['uo'] = ta.ULTOSC(dataframe)
+
+        # # Commodity Channel Index: values [Oversold:-100, Overbought:100]
         # dataframe['cci'] = ta.CCI(dataframe)
 
-        # MACD
-        macd = ta.MACD(dataframe)
-        dataframe['macd'] = macd['macd']
-        dataframe['macdsignal'] = macd['macdsignal']
-        dataframe['macdhist'] = macd['macdhist']
-
-        # MFI
-        dataframe['mfi'] = ta.MFI(dataframe)
-
-        # # Minus Directional Indicator / Movement
-        # dataframe['minus_dm'] = ta.MINUS_DM(dataframe)
-        # dataframe['minus_di'] = ta.MINUS_DI(dataframe)
-
-        # # Plus Directional Indicator / Movement
-        # dataframe['plus_dm'] = ta.PLUS_DM(dataframe)
-        # dataframe['plus_di'] = ta.PLUS_DI(dataframe)
-        # dataframe['minus_di'] = ta.MINUS_DI(dataframe)
-
-        # # ROC
-        # dataframe['roc'] = ta.ROC(dataframe)
+        # RSI
+        dataframe['rsi'] = ta.RSI(dataframe)
 
-        # # Inverse Fisher transform on RSI, values [-1.0, 1.0] (https://goo.gl/2JGGoy)
+        # # Inverse Fisher transform on RSI: values [-1.0, 1.0] (https://goo.gl/2JGGoy)
         # rsi = 0.1 * (dataframe['rsi'] - 50)
         # dataframe['fisher_rsi'] = (np.exp(2 * rsi) - 1) / (np.exp(2 * rsi) + 1)
 
-        # # Inverse Fisher transform on RSI normalized, value [0.0, 100.0] (https://goo.gl/2JGGoy)
+        # # Inverse Fisher transform on RSI normalized: values [0.0, 100.0] (https://goo.gl/2JGGoy)
         # dataframe['fisher_rsi_norma'] = 50 * (dataframe['fisher_rsi'] + 1)
 
-        # # Stoch
+        # # Stochastic Slow
         # stoch = ta.STOCH(dataframe)
         # dataframe['slowd'] = stoch['slowd']
         # dataframe['slowk'] = stoch['slowk']
 
-        # Stoch fast
+        # Stochastic Fast
         stoch_fast = ta.STOCHF(dataframe)
         dataframe['fastd'] = stoch_fast['fastd']
         dataframe['fastk'] = stoch_fast['fastk']
 
-        # # Stoch RSI
+        # # Stochastic RSI
         # stoch_rsi = ta.STOCHRSI(dataframe)
         # dataframe['fastd_rsi'] = stoch_rsi['fastd']
         # dataframe['fastk_rsi'] = stoch_rsi['fastk']
```

Comment:
Based on some recent acivity in #sharing-strategies on slack, i think we should add a warning to this indicator - as it's calculation is ... not in line what you see on exchange / tradingview (not to say ta-lib calculates it wrongly).

The slack-channel also includes a proposed new calculation method ... so i guess we could have that in technical (or evaluate how finta calculates stoch_rsi).

I'll however merge this anyway as that will require seperate evaluation and is just a "random" spot  while reviewing this.